packages/base/src/types.ts

```diff
@@ -18,7 +18,17 @@ type ClassMap = { [x: string] : ClassMapValue | ClassMap };
 
 type PassiveEventListenerObject = EventListenerObject & { passive: boolean };
 
+type AccessibilityInfo = {
+	role?: string, // The WAI-ARIA role of the component.
+	type?: string, // The component type (could be the component name), used when several components share same role (f.e. Select and ComboBox both have role="combobox").
```

Comment:
type should be translatable, please add it in the comment

Reply:
Recently, we had request where string aria attributes should be lowercase written. Maybe we should extend this somehow to require lowercase strings